Source/WebCore/rendering/RenderLayerCompositor.cpp

```diff
@@ -3489,7 +3489,7 @@ bool RenderLayerCompositor::requiresCompositingForBackfaceVisibility(RenderLayer
 
 bool RenderLayerCompositor::requiresCompositingForViewTransition(RenderLayerModelObject& renderer) const
 {
-    return renderer.capturedInViewTransition() || renderer.isViewTransitionPseudo();
+    return (renderer.document().activeViewTransition() && renderer.style().viewTransitionName()) || renderer.isViewTransitionPseudo();
```

Comment:
Do we need this change? Seems like the we shouldn't need it to have a compositing layer until the VT is actually running.

Reply:
I run into this assert if I don't do this change: `willBeComposited == needsToBeComposited(layer, queryData)`